content/research/Can AI Models Predict What Youll Say Next.md

```diff
@@ -0,0 +1,161 @@
+
+1. **No Context**: Models only received the immediate conversation snippet and the four options.
+2. **Raw Context**: Models received the conversation snippet plus the previous 50 or 100 messages from the Discord history (unfiltered and unprocessed).
+3. **Summary Context**: Models received the conversation snippet plus a generated personality profile of the target user, created by processing the previous 50 or 100 messages through Llama 3.3 70B.
+
+This design allowed us to compare whether any context provides useful signals for predicting social behavior, and whether a summary can provide results comparable to the full context.
+
+### Experimental Setup
+
+We tested a wide range of models including:
+- Claude 3.7 Sonnet, Claude 3.5 Sonnet, Claude 3.5 Haiku
+- GPT-4.5, GPT-4o, GPT-4o Mini, O-1, O-3 Mini
+- Google's Gemini 2.0 Flash
+- Meta's Llama 3.3 70B Instruct
+- Nous Research's Hermes 3 (405B and 70B variants)
+- DeepSeek models (Chat and R1)
+
+For each model and context mode combination, we ran three trials with different random seeds to control for position bias in option selection. Ideally we would have run more trials, but we wanted to constrain the compute needed for this experiment.
+
+## Results and Discussion
+
+![Figure 1: Model performance across different context modes](VSR_results.png)
+
+Our evaluation produced several notable results:
+
+### Context Helps Regardless of Form
+
+Additional context helps models predict social behavior, whether that context is provided as raw conversation history or as a processed summary. Moving from no context to either raw or summary context yielded substantial improvements for virtually all models tested. This confirms what might seem intuitive: knowing more about someone helps predict what they might say next.
+
+### Efficient Context Processing Works
+
+What's particularly significant is that injecting pre-processed summaries of user context works as well as or better than providing raw context for most models. This has important implications for system design:
+
+1. The summaries contain far fewer tokens than raw context (approximately one paragraph versus potentially thousands of tokens)
+2. The summarization can be done once with a smaller, cheaper model
+3. The resulting performance gains are substantial compared to no-context baselines, and in some cases even better than providing the full context
+
+This supports a core [thesis](https://blog.plasticlabs.ai/blog/Theory-of-Mind-Is-All-You-Need) behind Honcho: ambient processing of user context to generate compressed representations can improve model performance while keeping inference costs manageable. Rather than injecting massive amounts of data into the context window, models can achieve better results with distilled personality profiles.
```

Comment:
update "thesis" to a backlined note, then it'll preview on hover... in fact maybe backlink the entire "thesis behind Honcho" phrase, it fits (i think this would look like [[blog/Theory of Mind Is All You Need|thesis behind Honcho]])